app/assets/stylesheets/darkswarm/shop_tabs.css.scss

```diff
@@ -76,20 +76,17 @@
   // content revealed in accordion
 
   .page-view {
-    margin-bottom: 5em;
+    border-bottom: 5em solid $grey-080;
```

Comment:
the upper part of the background below this image was margin (cant change background color of margin) now it's border (can change background color of border)
![image](https://user-images.githubusercontent.com/1640378/79683595-2980e200-8223-11ea-891d-e96b59878d15.png)